### PR TITLE
Develop v3

### PR DESCRIPTION
**Pull request by Reachakhil**

### CODE REVIEW

File: application/api/connectiondetail.py [hunk 1]
```diff
@@ -41,23 +26,95 @@ def post(self, session):
...
+        """
+        try:
+
+            db_connection_detail = reqparse.RequestParser()
+            db_connection_detail.add_argument('project_id', required=False,
+                                              type=int,
+                                              location='args')
+
+            project_id = db_connection_detail.parse_args()
+            project_obj = Project.query.filter_by(
```
Comment (lalithbachimanchi) [inline application/api/connectiondetail.py]: When you are querying any DB record, make sure you also check is_deleted is False. For now we dont have delete option, but in future, we may need to add this condition. Its better we take care of it now itself.

File: application/common/constants.py [hunk 2]
```diff
@@ -52,6 +52,8 @@ class APIMessages:
 
     SOURCE = "source"
     DESTINATION = "destination"
+    PROJECT_NOT_EXIST = "Project does not exists"
```
Comment (lalithbachimanchi) [inline application/common/constants.py]: Project does not exist should be the message

File: application/api/connectiondetail.py [hunk 3]
```diff
@@ -41,23 +26,95 @@ def post(self, session):
...
                                 {'error_log': str(e)})
+
+
+class DbConnection(Resource):
+    """
+    Class will be a get call to give all the db_connection_ids
+     associated with the project_id
+    """
+
+    def get(self):
```
Comment (lalithbachimanchi) [inline application/api/connectiondetail.py]: is token not required for this api call?

File: application/api/connectiondetail.py [hunk 4]
```diff
@@ -41,23 +26,95 @@ def post(self, session):
...
+                                STATUS_SERVER_ERROR)
+
+
+class CaseDetails(Resource):
+    """
+    Class will take all the case_ids associated with a particular
+    test_Suite_id
+    """
+
+    def get(self):
```
Comment (lalithbachimanchi) [inline application/api/connectiondetail.py]: is token not required for this api call?

File: application/helper/returnallsuites.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 from application.model.models import TestSuite
 
 
-def return_all_suites(user_id):
+def return_all_suites(project_id):
     """
     Method that returns all the suite, case details associated with the user.
     Args:
```
Comment (lalithbachimanchi) [inline application/helper/returnallsuites.py]: docstring is not proper, describe what project_id is for and returns what

File: application/api/connectiondetail.py (hunk 1 above)
Comment (HarshadKavathiya) [reply application/api/connectiondetail.py]: For now, we are not validating this check. We will add it later.

File: application/helper/returnallsuites.py (hunk 5 above)
Comment (HarshadKavathiya) [reply application/helper/returnallsuites.py]: For now, we are not validating this check. We will add it later.

**Pull request merged**